cmd/manager/main.go

```diff
@@ -86,6 +87,10 @@ func main() {
 		log.Error(err, "unable to add OpenShift route APIs to scheme")
 		os.Exit(1)
 	}
+	if err := operatorv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "unable to add OpenShift operator APIs to scheme")
```

Comment:
Is this only available in OCP 4?

Reply:
Yes. This section will only run on the destination cluster which is guaranteed to be OCP 4.x.

Reply:
@djwhatle is there a concern putting OCP4.x specific apis to the manager scheme?

Reply:
Right now our controller works on OCP 3.x and 4.x, so this seems like it would be a breaking change.

Reply:
@djwhatle @dymurray Is this running on the destination cluster or the controller cluster? Controller can definitely be 3.x. Also, at one point things still had to work with dest being 3.x. Is that no longer the case? Is it now acceptable for a 3.x dest cluster to break completely?